### ISSUE TITLE
Support for encrypted references

### ISSUE THREAD

**Issue by agazso** [ethersphere/swarm-cid-js#10]
It would be good to have support for encrypted references.

**Comment by AuHau** [comment]
Agree, the reason why they were not implemented is because they won't fit for the bzz.link subdomain length constrain, but in other scopes they would make sense. But for the bzz.link usecase consumers of this library will have to check and disallow encrypted references for bzz.link...
